MeesGame/Gameplay/GameObjects/Player.cs

```diff
@@ -129,6 +133,31 @@ public bool HasKey()
             }
             return false;
         }
+
+        public void walkAnimation(string spritesheet)
+        {
+            Stopwatch time = new Stopwatch();
+            time.Start();
+            
+            Thread animationThread = new Thread(() => {
```

Comment:
I thought we agreed to avoid threading as much as possible. This doesn't look like a use case that justifies it.